src/rules/ig.toml

```diff
@@ -0,0 +1,23 @@
+min_trimmed_length = 3
+min_word_count = 3
+max_word_count = 14
```

Comment:
It seems to me that sentences have a rather large number of words. Not knowing the language I have no idea if those are just generally long sentences, but it at least seems that words generally are shorter in Igbo.

How many words on average can you speak out load in normal pace within around 5 seconds? How many in 7 seconds?

Bumping up the number here might make sense depending on that.